bcl2fastq/index_summary.py

```diff
@@ -5,100 +5,133 @@
-            #Bar graph with defaults set, no change in colour etc
+    # Bar graph with defaults set, no change in colour etc
     html.Div([
         html.Div(
             [dcc.Graph(id='known_unknown_pie'),
              dcc.Textarea(
                  id='known_fraction',
                  style={'width': '100%'},
                  readOnly=True,
+                 # This is the textbox at the bottom, hover over to see title
                  title=(
                      "Assumptions are made about which indexes are known "
                      "or unknown. This is due to multiple bcl2fastq analyses "
                      "being used on one run. This number should be 100%."
                  ),
-                        #This is the textbox at the bottom, hover over to see title
+
              )],
             style={'width': '24%', 'display': 'inline-block', }
         ),
-                 #Pie chart, by default is set to left with a width of 24% of secreen. 
         html.Div(
             [dcc.Graph(id='unknown_index_bar')],
             style={'width': '76%', 'display': 'inline-block', 'float': 'center'}
         ),
     ]),
+    #           Sample_run_hidden holds json split format of "Known"
+    #               columns: "FlowCell","Index1","Index2","LIMS IUS SWID","LaneClusterPF","LaneClusterRaw",
+    #                   "LaneNumber","LaneYield","QualityScoreSum","ReadNumber","Run","RunNumber","SampleID",
+    #                   "SampleName","SampleNumberReads","SampleYield","TrimmedBases","Yield","YieldQ30"
+    #               Index: 6761 - 8307
+
+    #           pruned_unknown_hidden holds json split format of "unknown"
+    #               columns:  "Count","LaneNumber","Index1","Index2","Run","LIMS IUS SWID"
+    #               Index: 218463 - 227804
+
     html.Div(id='sample_run_hidden', style={'display': 'none'}),
     html.Div(id='pruned_unknown_hidden', style={'display': 'none'}),
-            #html.div is just a way of dividing the HTML into meaningful groups
-            #Within the main layout division there are 4 groups: 
-            #           1. the section with the 100% bar graph
-            #           2.  Nested subdivision of: 
-            #                   a) pie graph at 24%
-            #                   b) bar graph at 76%
-            #           3. Sample Run hidden
-            #           4. Pruned  Unknown Hidden
-            #Note: span can be used for inline setting
+
 ])
 
 try:
-    from app import app 
-except ModuleNotFoundError: 
+    from app import app
+except ModuleNotFoundError:
     import dash
-    app = dash.Dash(__name__) 
+
+    app = dash.Dash(__name__)
     app.layout = layout
 
 
 @app.callback(
+
     dep.Output('sample_run_hidden', 'children'),
     [dep.Input('run_select', 'value')]
-#           Whenever input(drop down) changes, app.callback re-runs and updates     
+
 )
-def update_sample_run_hidden(run_alias):        
+#           Replace run variable iteratively
```

Comment:
Can this be incorporated into the docstring below, so there's only one place where the method is documented?